.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
could you remove the .vscode directory with `rm -r --cache .vscode` and then add `/.vscode` to the `.gitignore` file?

Reply:
Yep, will make the changes